packages/edit-post/src/components/block-manager/index.js

```diff
@@ -92,15 +87,15 @@ function BlockManager( {
 					<BlockManagerCategory
 						key={ category.slug }
 						title={ category.title }
-						blockTypes={ filter( blockTypes, {
-							category: category.slug,
-						} ) }
+						blockTypes={ blockTypes.filter(
```

Comment:
`blockTypes` will always be an array:

https://github.com/WordPress/gutenberg/blob/93c5efd7ebbebf0ae73247866e5f12589887ede0/packages/blocks/src/store/selectors.js#L71